src/commands/getex.json

```diff
@@ -0,0 +1,75 @@
+{
+    "GETEX": {
+        "summary": "Get the value of a key and optionally set its expiration",
+        "complexity": "O(1)",
+        "group": "string",
+        "since": "6.2.0",
+        "arity": -2,
+        "function": "getexCommand",
+        "command_flags": [
+            "WRITE",
+            "FAST"
+        ],
+        "acl_categories": [
+            "STRING"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "WRITE"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
+                "find_keys": {
+                    "range": {
+                        "lastkey": 0,
+                        "step": 1,
+                        "limit": 0
+                    }
+                }
+            }
+        ],
+        "arguments": [
+            {
+                "name": "key",
+                "type": "key",
+                "key_spec_index": 0
+            },
+            {
+                "name": "expiration",
+                "type": "oneof",
+                "optional": true,
+                "arguments": [
+                    {
+                        "name": "seconds",
+                        "type": "integer",
+                        "token": "EX"
+                    },
+                    {
+                        "name": "milliseconds",
+                        "type": "integer",
+                        "token": "PX"
+                    },
+                    {
```

Comment:
```
                    {
                        "name": "unix-time",
                        "type": "integer",
                        "token": "EXAT"
                    },
                    {
                        "name": "unix-time",
                        "type": "integer",
                        "token": "PXAT"
                    },
```
should we change it to?
```
                    {
                        "name": "timestamp",
                        "type": "unix-time",
                        "token": "EXAT"
                    },
                    {
                        "name": "milliseconds-timestamp",
                        "type": "unix-time",
                        "token": "PXAT"
                    },
```

Reply:
@enjoy-binbin if you find a problem or an inconsistency on an already merged PR, please just post a PR (rather than a comment in the closed one).

p.s. i'm not sure if milliseconds timestamp could be technically considered a "unix-time" but i see it already is in some fields, so let's at least be consistent.